notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>It would be really cool to explain these different RAG configurations earlier where you set them up. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Maybe I could link to the previous Advanced_RAG notebook instead, where they are explained ? Since the focus here is more evaluation.</p>

Reply:
Good idea!